package.json

```diff
@@ -5,8 +5,9 @@
   "main": "server.js",
   "scripts": {
     "test": "jest",
-    "start": "node server.js",
-    "db": "mysql -u root -p"
+    "start": "nodemon server.js",
```

Comment:
I'm gonna change this back to node server.js and add make a "dev" script for nodemon server.js